package.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gazebo_ros_sonar_plugin</name>
+  <version>0.0.0</version>
+  <description>Sonar Package based on ROCKY package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="ivan.tarifa@fieb.org.br">ivan</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
+
+  <depend>uuv_descriptions</depend>
+  <depend>uuv_sensor_ros_plugins</depend>
+  <depend>uuv_gazebo_ros_plugins</depend>
+  <depend>gazebo</depend>
+  <depend>gazebo_plugins</depend>
+  <depend>cv_bridge</depend>
+
+  <exec_depend>gazebo_ros</exec_depend>
+  <exec_depend>opencv3</exec_depend>
+
+  <test_depend>rosunit</test_depend>
+  <test_depend>rostest</test_depend>
+  
+  <build_depend>opencv3</build_depend>
+  <build_depend>roslint</build_depend>
```

Comment:
I think this package also depends on `gazebo_dev` to build...

Reply:
Yes, fixed